scripts/time_plot.py

```diff
@@ -1497,6 +1497,69 @@ def handle_websock_event(handlerkey,*args):
                 for thishandler in websockrequest_username.keys():
                     if (websockrequest_username[thishandler]==username):
                         send_websock_cmd('ApplyViewLayout('+'["'+'","'.join([fig['figtype'] for fig in html_viewsettings[username]])+'"]'+','+str(html_layoutsettings[username]['ncols'])+')',thishandler)
+        elif (args[0].startswith('holo')):#accepts 'holo', 'holo 3' where 3 is m003 as explicit reference antenna, else first track antenna is chosen as reference antenna
+            logger.info(repr(args))
+            antnumbers=[int(antnumberstr[1:]) for antnumberstr in telstate_antenna_mask]#determine all available inputs
```

Comment:
Those are antennas, not inputs.

Reply:
Well, they could also be d000 for example for CBF streams, but i guess they are not m001h.

Reply:
d000 wouldn't be in telstate_antenna_mask.

Reply:
but s000 would be for ska antennas. There were cases in the past that signals like d000 existed. Whether that is the case now or not, the code that deciphers/maps signal names should be able to deal with it.

Reply:
s000 is still an antenna. I'm just suggesting
```suggestion
            antnumbers=[int(antnumberstr[1:]) for antnumberstr in telstate_antenna_mask]#determine all available antennas
```